js/Formulary-validation.js

```diff
@@ -0,0 +1,38 @@
+const formulary = document.querySelector('#data');
+const emailInput = document.querySelector('#emailfield');
+const pemail = document.querySelector('.pemail');
+const send = document.querySelector('.pbutton');
+const Nosend = document.querySelector('.pbuttonNo');
+const expre = {
+  regexemail: /^[a-z0-9_.+-]+@[a-z0-9-]+\.[a-z0-9-.]+$/,
```

Comment:
- You did good implementing the form validation, but according to the instructions you are to display the error message only when the user tries submitting the form with Emails that includes Uppercase letters but on your validation, the error pops up as some as I start typing been if what I type is a valid input. Please try to fix this. 

Note: 
- The contact form contains at least one validation checking if the email is written in all lowercase:
   * If a user submits the contact form with the email field containing uppercase characters an error message should show up.
   * If a user submits the contact form with the email field containing only lowercase characters the form is sent normally.
   *  The error message should show up close to the "submit" button.
- The form "submit" or the submit button/input "click" event is used to attach the function checking the email format.

#####  VideoTesting of your design 

https://user-images.githubusercontent.com/58771507/175768968-63a878fd-0532-4ab3-8a8f-bcc9a81df8b6.mp4

Reply:
Thank you for your review but I do not agree with you, but it is a warning, if you put the complete email it will be removed, that is why this is the example of the email and leads to a more effective conclusion than waiting to have to send the form

Reply:
I kindly ask you to review more carefully and with valid and invalid examples

Reply:
Hi @SamuelTTW , What I am saying is the message should show close to the button when the user tries to submit, Telling the user to enter the email in lowercase, not to show the form not submitted as in your case. Please try to fix that, as learning to follow requirements is a needed skill. After the approval, you can implement extra.